Samples/VShadowVolumeShadowCopy/cpp/shadow.cpp

```diff
@@ -674,6 +674,27 @@ int CommandLineParser::MainRoutine(vector<wstring> arguments)
 
 #endif
 
+
+        // Perform a support check
+        if (MatchArgument(arguments[argIndex], L"isup")) //wsSnapIdDest
+        {
+            ft.WriteLine(L"(Started support check)");
+
+
+            // Initialize the VSS client
+            m_vssClient.Initialize(VSS_CTX_ALL);
+
+            WCHAR wszVolumePathName[MAX_PATH];
+            BOOL supported = TRUE;
+            BOOL bWorked = ::GetVolumePathName(arguments[1].c_str(), wszVolumePathName, MAX_PATH);
+
+
+            HRESULT is_supported_result = m_vssClient.IsVolumeSupported(wszVolumePathName, &supported);
```

Comment:
Shouldn't you check `is_supported_result` and return non-zero code in case of error?

Reply:
Nope, i need to check if this function returns VSS_E_NESTED_VOLUME_LIMIT